app/lib/mini-course.cjsx

```diff
@@ -107,6 +109,8 @@ module.exports = React.createClass
     optOut: false
 
   componentDidMount: ->
+    @props.geordi.makeHandler 'tutorial-completion'
```

Comment:
I'm a bit hesitant to say this, but I _don't_ think this is doing anything here. If you look at other instances of PFE where we use `geordi.makeHandler`, it is usually attached to a component and later called/bound with a click handler. Perhaps @eatyourgreens can confirm this.